idaes/power_generation/unit_models/helm/splitter.py

```diff
@@ -158,12 +158,12 @@ def sum_split(b, t):
         @self.Constraint(time, self.outlet_list, doc="Pressure constraint")
         def pressure_eqn(b, t, o):
             o_block = getattr(self, "{}_state".format(o))
-            return self.mixed_state[t].pressure == o_block[t].pressure
+            return self.mixed_state[t].pressure*1e-6 == o_block[t].pressure*1e-6
```

Comment:
We should really use the scaling tools to do stuff like this, but it's okay for now.  Probably should revisit it at some point.